lib/omnibus/packagers/fastmsi.rb

```diff
@@ -0,0 +1,654 @@
+#
+# Copyright 2015 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'pathname'
+
```

Comment:
Can you leave a not at the top of packages/msi and this file, saying that you copy-pastad the code for the purposes of an experiment and that fixes need to go into both of them.  Hopefully, this will succeed and we can kill the original msi packager completely.